README.md

```diff
@@ -1,3 +1,5 @@
 Hey!
 
-It worked!
\ No newline at end of file
+It worked!
```

Comment:
I think it's great that it worked, but please change it to not have exclamation point